core/block_svg.js

```diff
@@ -594,15 +595,15 @@ Blockly.BlockSvg.prototype.snapToGrid = function() {
   if (this.isInFlyout) {
     return;  // Don't move blocks around in a flyout.
   }
-  var grid = this.workspace.getGrid();
+  const grid = this.workspace.getGrid();
   if (!grid || !grid.shouldSnap()) {
     return;  // Config says no snapping.
   }
-  var spacing = grid.getSpacing();
-  var half = spacing / 2;
-  var xy = this.getRelativeToSurfaceXY();
-  var dx = Math.round((xy.x - half) / spacing) * spacing + half - xy.x;
-  var dy = Math.round((xy.y - half) / spacing) * spacing + half - xy.y;
+  const spacing = grid.getSpacing();
+  const half = spacing / 2;
+  const xy = this.getRelativeToSurfaceXY();
+  let dx = Math.round((xy.x - half) / spacing) * spacing + half - xy.x;
+  let dy = Math.round((xy.y - half) / spacing) * spacing + half - xy.y;
   dx = Math.round(dx);
   dy = Math.round(dy);
```

Comment:
Could you reformat this so these can be const?

Reply:
Done